### DISCUSSION TITLE
MenuKeyboard

### DISCUSSION THREAD

**Discussion by ranch-hands** [kianahmadian/python-bale-bot#67] [Q&A] [answered]
Hi Dear Authors:
how can we make a menu for a bot to show the list of facilities?
 Thank you very much.

**Comment by kian-ahmadian** [comment, marked as answer]
Hi @ranch-hands, if you want to use MenuKeyboard, you can read [examples](https://github.com/python-bale-bot/python-bale-bot/blob/master/examples/inline_markup.py#L21) & [documents](https://docs.python-bale-bot.ir/en/stable/bale.ui.menukeyboardmarkup.html) about that.
